packages/hash/api/src/graphql/resolvers/account/accounts.ts

```diff
@@ -1,8 +1,8 @@
-import { Resolver } from "../../apiTypes.gen";
+import { ResolverFn } from "../../apiTypes.gen";
 import { GraphQLContext } from "../../context";
 import { UnresolvedGQLEntity, Account } from "../../../model";
 
-export const accounts: Resolver<
+export const accounts: ResolverFn<
```

Comment:
New autogenerated types:

```
export type ResolverWithResolve<TResult, TParent, TContext, TArgs> = {
  resolve: ResolverFn<TResult, TParent, TContext, TArgs>;
};
export type Resolver<TResult, TParent = {}, TContext = {}, TArgs = {}> =
  | ResolverFn<TResult, TParent, TContext, TArgs>
  | ResolverWithResolve<TResult, TParent, TContext, TArgs>;

export type ResolverFn<TResult, TParent, TContext, TArgs> = (
  parent: TParent,
  args: TArgs,
  context: TContext,
  info: GraphQLResolveInfo,
) => Promise<TResult> | TResult;
```

If we keep using `Resolver`, calling `myResolver()` results an error as the variable is not necessarily callable. To avoid this complexity caused by `ResolverWithResolve`, I've ended up using `ResolveFn` everywhere.